code/modules/cargo/exports/parts.dm

```diff
@@ -110,7 +110,7 @@
 	include_subtypes = FALSE
 
 /datum/export/glasswork_lens
-	cost = 1800
+	cost = 1600
```

Comment:
id suggest 1200 but thats me

Reply:
I just subtracting the silk """"rarity"""" from its price, since cloth is readily available if you have a wirecutter and a bandage or bedsheet.